src/scales/scale.category.js

```diff
@@ -63,17 +63,28 @@ module.exports = Scale.extend({
 		return me.ticks[index - me.minIndex];
 	},
 
-	// Used to get data value locations.  Value can either be an index or a numerical value
-	getPixelForValue: function(value, index, datasetIndex) {
+	_configure: function() {
 		var me = this;
 		var offset = me.options.offset;
+		var ticks = me.ticks;
+
+		Scale.prototype._configure.call(me);
 
-		// 1 is added because we need the length but we have the indexes
-		var offsetAmt = Math.max(me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1), 1);
+		if (!me.isHorizontal()) {
+			// For backward compatibility, vertical category scale reverse is inverted.
+			me._reversePixels = !me._reversePixels;
+		}
+
+		if (ticks) {
```

Comment:
Can you reverse this? I think it would solve the Code Climate warning. This PR is the best I've seen for CC. You fixed 11 issues! :-)

```
if (!ticks) {
  return;
}
me._startValue = me.minIndex - (offset ? 0.5 : 0);
me._valueRange = Math.max(ticks.length - (offset ? 0 : 1), 1);
```

Reply:
Done